overrides.nix

```diff
@@ -1638,5 +1638,50 @@ self: super:
     }
   );
 
+  wxpython = super.wxpython.overridePythonAttrs (old: rec{
+    DOXYGEN = "${pkgs.doxygen}/bin/doxygen";
+
+    localPython = self.python.withPackages (ps: with ps; [
```

Comment:
Use a `let ... in` instead and remove the `rec`. Without this my initial thought was that `localPython` was some special python package attribute, which is confusing.

Reply:
Ok... Done! Thanks!